src/main/java/ecommerce/controller/OrderController.java

```diff
@@ -0,0 +1,154 @@
+package ecommerce.controller;
+
+import ecommerce.cache.LruCacheService;
+import ecommerce.dao.OrderDao;
+import ecommerce.entity.Order;
+import ecommerce.entity.Product;
+import ecommerce.exception.ApplicationRuntimeException;
+import ecommerce.exception.InvalidInputException;
+import ecommerce.service.OrderService;
+
+import java.sql.Connection;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.UUID;
+import java.util.logging.Logger;
+
+/**
+ * This class is used for managing place,delete and show order operation
+ *
+ * @author Akash gupta
+ */
+public class OrderController {
+    Scanner sc = new Scanner(System.in);
+    private static Logger logger = java.util.logging.Logger.getLogger(OrderController.class.getName());
+    OrderDao orderDao = new OrderDao();
+    OrderService orderService = new OrderService();
+
+    /**
+     * This class is used for placing orders
+     *
+     * @param custId customer id
+     * @param name   name of product
+     * @param con    connection
+     */
+    public void placeOrder(UUID custId, String name, Connection con) throws ApplicationRuntimeException {
+        Map<UUID, Product> menu = null;
+
```

Comment:
Menu should be fetched from product service as this a product service functionality